ui/elements/IconGroup/IconGroup.styles.ts

```diff
@@ -0,0 +1,11 @@
+import styled from 'styled-components';
+
+export const StyledIconGroup = styled.div<{
+    horizontal: boolean;
+    spacing: string;
+}>`
```

Comment:
props which are intended to be passed to a styled components BUT NOT THE ACTUAL HTML ELEMENT (like div) are supposed to be transient props

pls read about them here: https://medium.com/@probablyup/introducing-transient-props-f35fd5203e0c

would need to add $ prefix to them like this:
```suggestion
export const StyledIconGroup = styled.div<{
    $horizontal: boolean;
    $spacing: string;
}>`
```